packages/coin-support-starknet/src/operations/createAccounts/index.ts

```diff
@@ -0,0 +1,121 @@
+import { CreateAccountDeviceEvent } from '@cypherock/coin-support-interfaces';
+import {
+  GetAddressesFromDevice,
+  IMakeCreateAccountsObservableParams,
+  makeCreateAccountsObservable,
+} from '@cypherock/coin-support-utils';
+import { starknetCoinList } from '@cypherock/coins';
+import { AccountTypeMap } from '@cypherock/db-interfaces';
+import { GetPublicKeysEvent, StarknetApp } from '@cypherock/sdk-app-starknet';
+import { hexToUint8Array } from '@cypherock/sdk-utils';
+import { Observable } from 'rxjs';
+
+import { derivationPathSchemes } from './schemes';
+import {
+  ICreateStarknetAccountEvent,
+  ICreateStarknetAccountParams,
+  ICreatedStarknetAccount,
+} from './types';
+
+import * as services from '../../services';
+import { createApp, deriveAddress } from '../../utils';
+
+const DERIVATION_PATH_LIMIT = 15;
+const STRK_TOKEN_CONTRACT =
+  '0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d';
```

Comment:
This has been declared in multiple places, consdier moving them to a common source